setup.cfg

```diff
@@ -10,3 +10,22 @@ include_trailing_comma=True
 force_grid_wrap=0
 use_parentheses=True
 line_length=88
+
+[check-manifest]
```

Comment:
will also need to add `check-manifest` hook in `[.pre-commit-config.yaml](https://github.com/capitalone/DataProfiler/blob/main/.pre-commit-config.yaml)`